tests-build/tests/macros_clippy.rs

```diff
@@ -0,0 +1,25 @@
+#[cfg(feature = "full")]
+#[tokio::test]
+async fn test_with_semicolon_without_return_type() {
+    #![deny(clippy::semicolon_if_nothing_returned)]
+
+    dbg!(0);
+}
+
+#[cfg(feature = "full")]
+#[tokio::test]
+async fn test_without_semicolon_with_return_type_without() -> Result<(), ()> {
+    #![allow(clippy::needless_return)]
+    #![deny(clippy::semicolon_if_nothing_returned)]
+
+    return Ok(());
+}
+
+#[cfg(feature = "full")]
+#[tokio::test]
+async fn test_with_semicolon_with_return_type() -> Result<(), ()> {
+    #![allow(clippy::needless_return)]
+    #![deny(clippy::semicolon_if_nothing_returned)]
+
+    return Ok(());
+}
```

Comment:
These are identical?

Reply:
Removed both tests. I was thinking `retuern Ok (())` without a semicolon triggers clippy, but that was rustfmt. And before pushing, I made a `cargo fmt`, which added a semicolon to the first test. But in such a case, there was no point in these tests.